src/System.Private.Xml/src/System/Xml/Xsl/QIL/QilXmlReader.cs

```diff
@@ -368,6 +368,7 @@ private void EndElement()
                         MethodInfo mi = null;
                         QilName name = (QilName)list[0];
 
+#pragma warning disable 0436  //TODO: remove this when AppDomain.GetAssemblies() becomes available
```

Comment:
actually, it's already ocvered by your issue to remove the workaroudn stubs.

---
In reply to: [80328855](https://github.com/dotnet/corefx/pull/12017#discussion_r80328855) [](ancestors = 80328855)